samples/js/sample.js

```diff
@@ -14,40 +14,48 @@ CKEDITOR.config.height = 150;
 CKEDITOR.config.width = 'auto';
 
 var initSample = ( function() {
-	var wysiwygareaAvailable = isWysiwygareaAvailable(),
-		isBBCodeBuiltIn = !!CKEDITOR.plugins.get( 'bbcode' );
-
 	return function() {
 		var editorElement = CKEDITOR.document.getById( 'editor' );
 
-		// :(((
-		if ( isBBCodeBuiltIn ) {
+		// Sample content for bbcode.
+		if ( hasPlugin( 'bbcode' ) ) {
 			editorElement.setHtml(
 				'Hello world!\n\n' +
 				'I\'m an instance of [url=https://ckeditor.com]CKEditor[/url].'
 			);
 		}
 
 		// Depending on the wysiwygare plugin availability initialize classic or inline editor.
-		if ( wysiwygareaAvailable ) {
+		if ( hasPlugin( 'wysiwygarea' ) ) {
 			CKEDITOR.replace( 'editor' );
 		} else {
 			editorElement.setAttribute( 'contenteditable', 'true' );
-			CKEDITOR.inline( 'editor' );
+
+			var editor = CKEDITOR.inline( 'editor', {
+				on: {
+					instanceReady: function() {
+						// Without focus inline editor will not be seen by the users.
+						editor.focus();
+					}
+				}
+			} );
 
 			// TODO we can consider displaying some info box that
 			// without wysiwygarea the classic editor may not work.
 		}
 	};
 
-	function isWysiwygareaAvailable() {
+	function hasPlugin( name ) {
 		// If in development mode, then the wysiwygarea must be available.
 		// Split REV into two strings so builder does not replace it :D.
 		if ( CKEDITOR.revision == ( '%RE' + 'V%' ) ) {
-			return true;
+			var config = {};
+			CKEDITOR.editorConfig( config );
+
+			return config.plugins.split( ',' ).indexOf( name ) > -1;
 		}
 
-		return !!CKEDITOR.plugins.get( 'wysiwygarea' );
+		return Boolean( CKEDITOR.plugins.get( name ) );
```

Comment:
We do use double negation in our code style. Don't change it to Boolean constructor.

Reply:
TBO I cannot disagree more. In my opinion, it's better to be explicit, not implicit. It's more verbose and more performant (although performance is not an important statement here). I think we should improve our code if we can, not follow bad code styles. For me, double negation is as bad as a bitwise operator (ya, I was trying to smuggle this one into our code base 😭). However, I think it would be nice to hear our team opinion thus it may be not objective.

Reply:
I understand your point of view, but I think its more important to abide [code style guide](https://github.com/ckeditor/ckeditor-idiomatic.js)👮. According to it:

>All code in any code-base should look like a single person typed it, no matter how many people contributed.
>"Arguments over style are pointless. There should be a style guide, and you should follow it"

If you disagree with it, you should start with raising discussion among team. And until we decide to change it please follow current rules. Note that we have only 5 occurrences of phrase 'Boolean( ' on our `master` branch and 2 of them are coming for your commits.

Reply:
In the big part of our code, we also have a single line if statements without brackets. What about it?

Reply:
I believe that this issue is not so much important that we could firstly resolve the issue with code styles.

Reply:
It seems like our team agrees that we should follow already defined rules. I will refactor this code soon.